install/0000_80_machine-config-operator_02_containerruntimeconfig.crd.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: containerruntimeconfigs.machineconfiguration.openshift.io
```

Comment:
I'm a bit confused, isn't this file mostly just a copy of 01_containerruntineconfig.crd.yaml? Why is this split out into a second file?

Reply:
the operative lines are `release.openshift.io/feature-gate: TechPreviewNoUpgrade` on line 11 and `pattern: ^$|^(runc|crun)$` on line 89. Basically, this crd enables `crun` as an option, but only when tech preview is enabled on the node

Reply:
Right, I think I might be confused here. I was under the impression the `02` crd will simply overwrite `01` regardless of whether the feature gate itself is active or not. It looks like from

https://github.com/openshift/machine-config-operator/pull/3295#issuecomment-1222772121

That is not the case and it actually deploys a different CRD based on the conditional.

Reply:
CVO magic :smiley: